examples/Playground.hs

```diff
@@ -1,11 +1,17 @@
 {-# OPTIONS -Wno-unused-top-binds #-}
+{-# LANGUAGE DeriveGeneric  #-}
```

Comment:
`DeriveGeneric` is already enabled in the `default-extensions`, so need to enable it in this module :slightly_smiling_face: